art/art_dic.py

```diff
@@ -46,7 +46,8 @@
                 "coffee": "c[_]",
                 "robot1": "d[ o_0 ]b",
                 "pig1": "^(*(oo)*)^",
-                "needle": "|==|iiii|>----- ",
+                "needle1": "|==|iiii|>----- ",
+                "needle2": "┣▇▇▇═─",
```

Comment:
I think it could be better if we change _needle2_ text art to _needle1_, it seems _needle2_ is more nature than _needle1_ and it should have priority when the user type "needle" for text art